src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/NotifyService.kt

```diff
@@ -199,27 +203,51 @@ class NotifyService(
     }
   }
 
-  @TimeServedConsiderations("If a COM is not present, where should this email be sent to?")
-  fun sendHardStopLicenceApprovedEmail(
+  fun sendReviewableLicenceApprovedEmail(
     emailAddress: String?,
     firstName: String,
     lastName: String,
     crn: String?,
     lsd: LocalDate?,
     licenceId: String,
+    prisonName: String,
+    isTimeServedLicence: Boolean = false,
   ) {
+    val isTimeServed = isTimeServedLogicEnabled && isTimeServedLicence
+    val templateId = if (isTimeServedLogicEnabled) reviewableLicenceApprovedTemplateId else hardStopLicenceApprovedTemplateId
     if (emailAddress == null || lsd == null) {
-      log.error("Notification failed (hardStopLicenceApproved) for licence $licenceId - email and CRD must be present")
+      val approvalType = if (isTimeServed) "timeServedLicenceApproved" else "hardStopLicenceApproved"
+      log.error(
+        "Notification failed ($approvalType) " +
+          "for licence $licenceId - email and CRD must be present",
+      )
+
       return
     }
-    val values: Map<String, String> = mapOf(
-      "firstName" to firstName,
-      "lastName" to lastName,
-      "crn" to crn!!,
-      "releaseDate" to lsd.format(dateFormat),
-    )
-    if (sendEmail(hardStopLicenceApprovedTemplateId, emailAddress, values)) {
-      log.info("Notification sent to $emailAddress HARD STOP LICENCE APPROVED for $licenceId $firstName $lastName")
+
+    val reasonForStandardLicence = if (isTimeServed) {
+      "this person was released immediately following sentencing having served time on remand"
+    } else {
+      "none was submitted in time for their final release checks"
+    }
+
+    val values = buildMap<String, String> {
+      put("firstName", firstName)
+      put("lastName", lastName)
+      put("crn", crn ?: "")
+      put("releaseDate", lsd.format(dateFormat))
+      if (isTimeServedLogicEnabled) {
+        put("reasonForStandardLicence", reasonForStandardLicence)
+        put("prisonName", prisonName)
+      }
+    }
+
+    if (sendEmail(templateId, emailAddress, values)) {
+      val licenceStatus = if (isTimeServed) "TIME SERVED" else "HARD STOP"
+      log.info(
+        "Notification sent to $emailAddress $licenceStatus LICENCE APPROVED " +
+          "for $licenceId $firstName $lastName",
+      )
```

Comment:
think it would be better to split this into two.. based on the number of conditionals in here it seems like it's trying to do two things - send two different emails!